Util/LegacyFormHelper.php

```diff
@@ -48,6 +48,6 @@ public static function getType($class)
 
     public static function isLegacy()
     {
-        return !method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix');
+        return false;
```

Comment:
All covered Symfony versions have this method, this class can probably be removed since we're never legacy.